includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1064,18 +1060,13 @@ protected function get_wizard_in_cart_payment_gateways() {
 				'enabled' => $can_stripe,
 				'featured' => true,
 			),
-			'braintree_paypal' => array(
-				'name'        => __( 'PayPal by Braintree', 'woocommerce' ),
-				'image'       => WC()->plugin_url() . '/assets/images/paypal-braintree.png',
-				'description' => $paypal_bt_description,
-				'repo-slug'   => 'woocommerce-gateway-paypal-powered-by-braintree',
-			),
 			'ppec_paypal' => array(
 				'name'        => __( 'PayPal Express Checkout', 'woocommerce' ),
 				'image'       => WC()->plugin_url() . '/assets/images/paypal.png',
 				'description' => $paypal_ec_description,
 				'class'       => 'checked',
 				'repo-slug'   => 'woocommerce-gateway-paypal-express-checkout',
+				'enabled' => true,
```

Comment:
@claudiulodro FYI, this was meant to supercede the logic in https://github.com/woocommerce/woocommerce/pull/17738/files#diff-f664362a33faa9d314f0582223d39a75R1160 — let me know if it was merged that way intentionally, but I can just fix in https://github.com/woocommerce/woocommerce/pull/18123 otherwise

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/18123/commits/4511c5ae1216193cbc3ad1db36704ded51acd6eb